core/field_multilineinput.js

```diff
@@ -70,19 +75,14 @@ Blockly.FieldMultilineInput.LINE_HEIGHT = 20;
 /**
  * Construct a FieldMultilineInput from a JSON arg object,
  * dereferencing any string table references.
- * @param {!Object} options A JSON object with options (text, class, and
- *                          spellcheck).
+ * @param {!Object} options A JSON object with options (text, and spellcheck).
  * @return {!Blockly.FieldMultilineInput} The new field instance.
  * @package
  * @nocollapse
  */
 Blockly.FieldMultilineInput.fromJson = function(options) {
   var text = Blockly.utils.replaceMessageReferences(options['text']);
-  var field = new Blockly.FieldMultilineInput(text);
-  if (typeof options['spellcheck'] === 'boolean') {
-    field.setSpellcheck(options['spellcheck']);
-  }
-  return field;
+  return new Blockly.FieldMultilineInput(text, null, options);
```

Comment:
Just FYI, and this affects all fields not just this one (I just haven't gotten around to fixing it up), but passing null here is wrong as the constructor is expecting the type ``Function=``, ie: ``Function|undefined``. Instead this can be changed to ``undefined``.